docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/bias_quantification.ipynb

```diff
@@ -0,0 +1,1387 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    Calculates absolute and relative error in the change in pre and post intervention
</pre><p>This isn't necessarily a helpful comment, but this sentence is pretty long; maybe could be broken up?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>